code/index.html

```diff
@@ -3,28 +3,198 @@
 
-    <!-- Signup form -->
+    <main>
+      <!-- Signup form -->
+      <section class="form">
+        <form action="http://httpbin.org/anything" method="POST">
+          <div class="form-title">
+            There is a solution to every behavioural issue. Trust us. 
+            We're dog behaviourists.
+            Get in touch for a consultation session!
+          </div>
+          
+          <!--OWNER'S INFO-->
+          <fieldset class="owner-info">
+          <legend>Information about owner</legend>
+          <div class="owner-info-div">
+            <label>
+              Your full name:
+              <input type="text" required name="fullName" class="input-owner">
+            </label>
+            <label>
+              E-mail:
+              <input type="email" required name="emailAddress" class="input-owner">
+            </label>
+            <label>
+              Phone number:
+              <input type="tel" name="phoneNumber" class="input-owner">
+            </label>
+          </div>
+          </fieldset>
+  
+          <!--DOG'S INFO-->
+          <fieldset class="dog-info">
+          <legend>About your dog</legend>
+            <div class="dog-info-div">
+              <label class="">
+                Dog's name:
+                <input type="text" required name="dogName">
+              </label>
+              <label>
+                Breed:
+                <input type="text" required name="breed">
+              </label>
+              <!-- <label>
+                Dog's gender:
+                <input type="radio" required name="dogGender">male
+                <input type="radio" required name="dogGender">female
+              </label>
+              <label>
+                Spayed/Neutered:
+                <input type="radio" required name="spayedNeutered"> YES
+                <input type="radio" required name="spayedNeutered"> NO
+              </label>
+              <label>
+                Behavioural issue:
+                
+                <textarea name="behaviouralIssue" rows="" cols="">
+                </textarea>
+              </label> -->
+              <!--need div to get the radio buttons in a row not in a column-->
+              <div class="dogs-gender">
+                <label>Dog's gender:
+                  <input type="radio" required name="dogGender">male
+                  <input type="radio" required name="dogGender">female
+                </label>
+              </div>
+              <div class="spayed-neutered">
+                <label>
+                  Spayed/Neutered:
+                  <input type="radio" required name="spayedNeutered"> yes
+                  <input type="radio" required name="spayedNeutered"> no
+                </label>
+              </div>
+              <div class="issue">
+                <label>
+                  Behavioural issue:
+                  <div>
+                    <textarea name="behaviouralIssue">
+                    </textarea>
```

Comment:
So when we're using the textarea-tag we need to have the closing tag right after the opening tag, otherwise it will create a lot of empty space on your site. Like this:
```
<textarea name="behaviouralIssue"></textarea>
```